### ISSUE TITLE
[Feature Request] Text replacement: asterisk to bullet

### ISSUE THREAD

**Issue by RadRussianRus** [telegramdesktop/tdesktop#4810]
In Markdown, when we write `*[space]` in start of line, we're creating unordered list. With phone keyboard you can easily type this symbol.

So I suggest this text replacement: `[start of line]*[space]` to `[start of line]•[space]` (U+2022).

**Comment by Lonami** [comment]
You may want to look into userbots to edit your sent messages into the most fancy ways. Telegram Desktop uses a markdown-like syntax for formatting, but it isn't a (proper) Markdown input.

**Comment by RadRussianRus** [comment]
Yes, I know, and I'm not asking for full Markdown support. It is just a suggestion to make list formatting easier.

**Comment by RadRussianRus** [comment]
Let it stay for now.
